administrator/components/com_content/views/article/tmpl/edit.php

```diff
@@ -164,9 +164,9 @@
 			<?php echo JHtml::_('bootstrap.addTab', 'myTab', 'menulink', JText::_('COM_CONTENT_MENU_LINK', true)); ?>
 			<div class="row-fluid form-horizontal-desktop">
 				<div class="span6">
-					<?php echo $this->form->getControlGroup('menulink'); ?>
+					<?php echo $this->form->renderField('menulink'); ?>
 					<?php foreach ($this->form->getGroup('menulink') as $field) : ?>
-						<?php echo $field->getControlGroup(); ?>
+						<?php echo $field->renderField(); ?>
```

Comment:
Because when I was goinh through the documentation I saw that getcontrolgroup and getcontrogroups is deprecated...
Here is the link https://googleweblight.com/?lite_url=https://api.joomla.org/cms-3/classes/JForm.html&ei=IQ43aJM1&lc=en-IN&s=1&m=250&host=www.google.co.in&ts=1468233329&sig=AKOVD67zDQkNI7fwfy2BS0dHfM0pVWG54A

Should I ignore this?